clj/resources/static_content/js/run_overview.js

```diff
@@ -144,21 +140,23 @@ jQuery( function() { ( function( $$, $, undefined ) {
               enable: true,
               onShow: function(tip, node) {
                 tip.innerHTML = "";
+                ctx = ($$.run.lastRefreshData && $$.util.string.isNotEmpty($$.run.lastRefreshData))? $$.run.lastRefreshData : $(":root");
```

Comment:
I'd refactor this line as:

``` javascript
var ctx = $$.util.string.isEmpty($$.run.lastRefreshData) ? $(":root") : $$.run.lastRefreshData;
```

1. Add `var` to avoid a global variable.

1. The  `string.isEmpty()` and `string.isNotEmpty()` util functions already take into account the fact that the argument is `undefined`:

```javascript
SlipStream.util.string.isEmpty("asdf")
// false
SlipStream.util.string.isEmpty("")
// => true
SlipStream.util.string.isEmpty()
// => true
SlipStream.util.string.isEmpty(undefined)
// => true
```

@schaubl 
Please commit this change if you agree. Cheers.